### PR TITLE
test: add test for airdrop.py

### PR DESCRIPTION
**Pull request by martinvibes**
hello @djeck1432 kindly review :) #141

### CODE REVIEW

File: web_app/contract_tools/airdrop.py [hunk 1]
```diff
@@ -49,9 +54,9 @@ def _validate_response(data: List[dict]) -> AirdropResponseModel:
...
             validated_item = AirdropItem(
                 amount=item["amount"],
-                proof=item["proof"],
+                proof=item["proof"],  # This is correct now as AirdropItem expects List[str]
                 is_claimed=item["is_claimed"],
                 recipient=item["recipient"]
             )
             validated_items.append(validated_item)
-        return AirdropResponseModel(airdrops=validated_items)
+        return AirdropResponseModel(airdrops=validated_items)
```
Comment (djeck1432) [inline web_app/contract_tools/airdrop.py:62]: add blank line in the end of the file
Screenshot 2024-10-30 at 7 58 10

File: web_app/tests/test_zklend_airdrop.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    async def test_underlying_contract_id_formatting(self, zklend_airdrop):
+        """Test correct formatting of underlying contract ID."""
+        
+        contract_id = "0x123456"
+        expected_underlying_id = "0x0123456"
+        zklend_airdrop.api.fetch.return_value = []
+
+        await zklend_airdrop.get_contract_airdrop(contract_id)
+
+        zklend_airdrop.api.fetch.assert_called_once_with(expected_underlying_id)
```
Comment (djeck1432) [inline web_app/tests/test_zklend_airdrop.py:125]: the same thing, add blank line int the end

**Review by djeck1432 — CHANGES_REQUESTED** [review]

**Comment by djeck1432** [comment]
@martinvibes Any updates?

**Comment by martinvibes** [comment]
@djeck1432 implemented requested change

**Pull request merged**